xpk.py

```diff
@@ -1107,6 +1328,8 @@ def run_gke_cluster_create_command(args) -> int:
       f' --cluster-version={args.gke_version} --location-policy=BALANCED'
       f' --machine-type={machine_type}'
       ' --scopes=storage-full,gke-default'
+      ' --enable-ip-alias '
+      f' --create-subnetwork name={args.cluster}-subnetwork '
```

Comment:
Which network will this subnetwork be created in? A project can have multiple networks.

Reply:
I will check that, currently I believe it uses the default network.

Reply:
What's the purpose of creating the subnetwork?

Reply:
This was added to Pathways cluster creation to mitigate this error, but is widely useful for all GKE cluster creation -
ERROR: (gcloud.container.clusters.create) ResponseError: code=400, message=This operation will exceed max secondary ranges per subnetwork (30) for subnet "default", consider reusing existing secondary ranges or use a different subnetwork.

Reply:
Moved this to be Pathways specific for now, but I think this might be widely useful.